docs/openapi-backend.yaml

```diff
@@ -0,0 +1,185 @@
+openapi: 3.0.3
+info:
+  title: IO Sign - Backend API
+  version: 0.0.1
+paths:
```

Comment:
maybe here should be added a security section and a base path https://github.com/pagopa/io-backend/blob/aa7d7c64c71b582d87f229b15a6e26c59c80a76c/api_backend.yaml#L6-L11
What do you think?

Reply:
Added. Check if the server is right

Reply:
the `basePath` should be `basePath: /api/v1/fci`